AntWars/Helper/CostCalculator.cs

```diff
@@ -71,9 +71,9 @@ public static int calculateCostCarry(int viewRange, int moveRangeFactor, int inv
             return calculate(viewRangeWithQuantifier, moveRangeFactorWithQuantifier, inventoryWithQuantifier);
         }
 
-        private static int calculate(double viewRangeWithQuantifier, double moveRangeFactorWithQuantifier, double inventoryWithQuantifier)
+        private static double calculate(double viewRangeWithQuantifier, double moveRangeFactorWithQuantifier, double inventoryWithQuantifier)
         {
-            return Convert.ToInt32(Math.Ceiling((viewRangeWithQuantifier + moveRangeFactorWithQuantifier + inventoryWithQuantifier) / 2));
+            return Math.Round(((viewRangeWithQuantifier + moveRangeFactorWithQuantifier + inventoryWithQuantifier) / 2), 2);
```

Comment:
@arndtphillip da ist eine ``(`` zuviel, kannst du ja in einem anderen issue mitfixen